iothub/service/src/Messaging/MessagesClient.cs

```diff
@@ -88,11 +88,16 @@ protected MessagesClient()
         /// public void OnConnectionLost(ErrorContext errorContext)
         /// {
         ///    // Add reconnection logic as needed
-        ///    Console.WriteLine("Messaging client connection lost")
+        ///    Console.WriteLine("Messaging client connection lost");
```

Comment:
How about just call openasync as an example?

Reply:
Sure, that works